dojson/contrib/marc21/utils.py

```diff
@@ -126,7 +126,22 @@ def split_blob(blob):
     raise StopIteration()
 
 
+def split_stream(stream):
+    """Split the stream using <record.*?>.*?</record> as pattern."""
+    buf = []
```

Comment:
But the current implementation is not transparent. `etree.tostring(record)` doesn't return the original content:
```
E       assert '<record>\n  ...d>\n</record>' == '<record>\n  <...d>\n</record>'
E         Skipping 324 identical leading characters in diff, use -v to show
E         - ="a">&lt;p&gt;Model definitions and data...&lt;/p&gt;</subfield>
E         ?           ^^^^                                   ^^^^
E         + ="a">&lt;p>Model definitions and data...&lt;/p></subfield>
E         ?           ^                                   ^
E             </datafield>
E             <datafield tag="540" ind1=" " ind2=" ">
E         -     <subfield code="u"/>
E         +     <subfield code="u"></subfield>
E         Detailed information truncated (30 more lines), use "-vv" to show
```